api/src/pcapi/routes/backoffice/venues/forms.py

```diff
@@ -67,6 +67,12 @@ class EditVenueForm(EditVirtualVenueForm):
         "Activité principale", choices=utils.choices_from_enum(offerers_models.VenueTypeCode)
     )
     is_permanent = fields.PCSwitchBooleanField("Lieu permanent")
+    acceslibre_url = fields.PCOptUrlField(
+        "Url chez acceslibre",
+        validators=[
+            fields.AcceslibreValidator(),
+        ],
+    )
```

Comment:
Pour poursuivre mon commentaire précédent, ici : 
```suggestion
    acceslibre_url = fields.PCOptStringField(
        "URL chez acceslibre",
        validators=(wtforms.validator.Optional(), wtforms.validator.URL(), AcceslibreValidator()),
    )
```